lib/rubyblok/helpers/storyblok_helper.rb

```diff
@@ -10,7 +10,13 @@ def rubyblok_content_tag(content)
   end
 
   def rubyblok_story_tag(slug)
-    content = get_story_via_api(slug)["content"].to_dot
+    content =
```

Comment:
apparently we should expose a public method
```
def get_story(slug)
  if use_cache?
    get_story_via_cache(slug)["content"].to_dot
  else
    get_story_via_api(slug)["content"].to_dot
  end
end
```

as in some cases we should fetch the story content in the controller instead of rendering the whole page via `rubyblok_story_tag(slug)`